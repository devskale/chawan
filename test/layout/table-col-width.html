<table>
<tr><td bgcolor=red>test</td></tr>
<tr><td bgcolor=red>a</td></tr>
</table>

<table>
<tr><td bgcolor=red style="width: 5ch">test</td></tr>
<tr><td bgcolor=red>a</td></tr>
</table>

<table>
<tr><td bgcolor=red>test</td></tr>
<tr><td bgcolor=red style="width: 5ch">a</td></tr>
</table>

<table>
<tr><td bgcolor=red style="width: 1ch">test</td></tr>
<tr><td bgcolor=red style="width: 5ch">a</td></tr>
</table>
