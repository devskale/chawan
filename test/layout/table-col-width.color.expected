 [48;2;255;0;0mtest[49m
 [48;2;255;0;0ma   [49m
 [48;2;255;0;0mtest [49m
 [48;2;255;0;0ma    [49m
 [48;2;255;0;0mtest [49m
 [48;2;255;0;0ma    [49m
 [48;2;255;0;0mtest [49m
 [48;2;255;0;0ma    [49m
